### ISSUE TITLE
Question: update pcu version

### ISSUE THREAD

**Issue by mathieupotier** [sermatec-opensource/sermatec-inverter#76]
Hi there,

My inverter currently running 6.1.6 and your read me now shows that it requires 6.1.9 to run.

Do we have any way to get the updated version ?
Tried to contact Sermatec service few month ago but no response from them.

Regards

**Comment by andreondra** [comment]
Hi,

maybe I worded it badly. It should work on all versions, the newest tested is the mentioned one. But I tested it also on 4.1.1 and works without problems. The script detects the inverter's version and parses data accordingly, so no worries 

As for the firmware update, I don't know whether the Sermatec will respond — I heard that they stopped selling the inverters all together. Maybe we can try contacting some installation company? But I try to keep required version as low as possible, so the updates shouldn't be needed.
